views/personal/address-edit.pug

```diff
@@ -21,7 +21,7 @@ block content
       h2 #{__('Your mailing address')}
 
       div
-        +textInput('Street number', ['w-1-4','d--ib'])(class='w-3-4', value=streetNumber, name='streetNumber')
+        +textInput('Street number', ['w-1-4','d--ib'])(class='w-3-4', value=streetNumber, name='streetNumber' autofocus=(!errors || errors && firstError.param === 'streetNumber' ? true : false))
```

Comment:
Can we put the `firstError` var in this template itself since it's the only time we use it?

Reply:
definitely could. I was thinking about in the future when there might be more forms with multiple inputs. But you're right that for now it's just this page

Reply:
I think maybe you deleted 'firstError' 🤔